util/auto_skip_compressor.h

```diff
@@ -87,4 +140,45 @@ class AutoSkipCompressorManager : public CompressionManagerWrapper {
       CompressionType preferred) override;
 };
 
+class CPUIOAwareCompressor : public Compressor {
```

Comment:
For the future, should we use MultiCompressorWrapper?

Reply:
Isn't "std::vector<std::vector<std::unique_ptr<Compressor>>> allcompressors_;" doing something  MulitCompressorWrapper does https://github.com/facebook/rocksdb/blob/820a30f0d23a79325b15cddd96f79f47178d5391/util/simple_mixed_compressor.h#L31?